plugins/pubIds/urn/locale/uk_UA/locale.xml

```diff
@@ -2,52 +2,71 @@
 <!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
 
 <!--
-  * plugins/pubIds/urn/locale/uk_UA/locale.xml
```

Comment:
You seem to have copied text from the DOI plugin. Please remove this file or update it according to the en_US locale of the URN plugin.